charts/portal/README.md

```diff
@@ -502,23 +502,22 @@ The following table lists the configured parameters of the MySQL Subchart - http
 | `mysql.primary.configuration`   | MySQL Primary configuration to be injected as ConfigMap	   | `see values.yaml` |
 
 
-## Nginx-Ingress
-The following table lists the configured parameters of the Nginx-Ingress Subchart - https://github.com/helm/charts/tree/master/stable/nginx-ingress
+## Ingress-Nginx
+The following table lists the configured parameters of the Ingress-Nginx Subchart - https://github.com/kubernetes/ingress-nginx/tree/main/charts/ingress-nginx
 
 This represents minimal configuration of the Chart, this can be disabled in favor of your own ingress controller in the ingress settings.
 IngressClass resources are supported since k8s >= 1.18 and required since k8s >= 1.19
 
 | Parameter                        | Description                               | Default                                                      |
 | -----------------------------    | -----------------------------------       | -----------------------------------------------------------  |
-| `nginx-ingress.podSecurityPolicy.enabled`                | Tell Nginx to read PSP   | `true` |
-| `nginx-ingress.rbac.create`                | Create & use RBAC resources   | `true` |
-| `nginx-ingress.controller.publishService.enabled`                | Enable Publish Service   | `true` |
-| `nginx-ingress.extraArgs.enable-ssl-passthrough`                | Enable SSL Passthrough   | `true` |
-| `nginx-ingress.deploymentLabels` | Ingress controller labels. Evaluated as a template | `{}` |
-| `nginx-ingress.ingressClassResource.name` | Creation of the IngressClass- with the name | `nginx` |
-| `nginx-ingress.ingressClassResource.enabled` | Creating the IngressClass- with the name specified, if this flag is enabled | `true` |
-| `nginx-ingress.ingressClassResource.default` | Creating the IngressClass- with the name specified, by default | `false` |
-| `nginx-ingress.ingressClassResource.controllerValue` | Creating the IngressClass- with the contollerValue specified | `k8s.io/ingress-nginx` |
+| `ingress-nginx.podSecurityPolicy.enabled`                | Tell Nginx to read PSP   | `true` |
+| `ingress-nginx.rbac.create`                | Create & use RBAC resources   | `true` |
+| `ingress-nginx.controller.publishService.enabled`                | Enable Publish Service   | `true` |
+| `ingress-nginx.controller.extraArgs.enable-ssl-passthrough`                | Enable SSL Passthrough   | `true` |
+| `ingress-nginx.controller.ingressClassResource.name` | Creation of the IngressClass- with the name | `nginx` |
+| `ingress-nginx.controller.ingressClassResource.enabled` | Creating the IngressClass- with the name specified, if this flag is enabled | `true` |
+| `ingress-nginx.controller.ingressClassResource.default` | Creating the IngressClass- with the name specified, by default | `false` |
```

Comment:
Write as ' conveys if this the default ingressClass for the cluster' and default value is false.

Reply:
addressed the review comment